Classes/RepresentationSerializer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Flownative\Sentry;
+
+/*
+ * This file is part of the Flownative.Sentry package.
+ *
+ * (c) Karsten Dambekalns, Flownative GmbH - www.flownative.com
```

Comment:
I think we should find some consistent copyright notice, as mentioning the author per file probably doesn't even have the desired legal effect.

I suggest that we either keep on using `(c) Robert Lemke, Flownative GmbH´ to give attribution to me being the original author or we use something more general.

Certainly not comparable, but just for inspiration, I looked at Phpunit, for example, which only mentions Sebastian as the original author despite the contributions by > 400 people.

I don't mean to diminish your contribution though, so if it's important to you, let's add more names.

Reply:
Whatever… I think the names are more or less useless TBH. But since you didn't create that file, it felt weird to put your name there. 🤷‍♂️